doc/components/dropdown.md

```diff
@@ -26,6 +26,11 @@ like `text="Click me" icon="icon-filter"`. As following you can see an example o
   </div>
 </div>
 
+**Disabled**
+As any common html element the dropdown can be disabled by adding the `disabled="1"` flag to it.
```

Comment:
It is not working. I described why somewhere in the document or in the code. The problem is that disabled will result in the property disabled: "" which will be false.